src/main.cpp

```diff
@@ -349,15 +360,13 @@ void ProxySQL_Main_init() {
 }
 
 
-
-
-
-
 void ProxySQL_Main_init_phase2___not_started() {
 	ProxySQL_Main_init_main_modules();
 	ProxySQL_Main_init_Admin_module();
 	GloMTH->print_version();
 
+	ProxySQL_Main_init_AlertRouter();
```

Comment:
I would move it from `ProxySQL_Main_init_phase2___not_started()` to `ProxySQL_Main_init_phase3___start_all()` .
It isn't a core module, so it can wait phase3